packages/blockchain-extension/extension/util/ExtensionUtil.ts

```diff
@@ -556,6 +556,15 @@ export class ExtensionUtil {
             }
         });
 
+        if (!process.env.REFRESH_PANELS || process.env.REFRESH_PANELS === 'true') {
+            setInterval(() => {
```

Comment:
can you not reuse the set interval function we have already

Reply:
Updated to use the TimerUtil.setInterval function - had to change it to handle multiple sets of commands and arguments.